### ISSUE TITLE
Detect multiline method calls by `Layout/MultilineAssignmentLayout`

### ISSUE THREAD

**Issue by fatkodima** [rubocop/rubocop#14311]
I would like to be able to disallow the code layout like
```ruby
foo = bar(
        1,
        2,
        3
      )
```

and enforce it to be
```ruby
foo = 
  bar(
    1,
    2,
    3
  )
```

For this we need to introduce a new value (named like `call`) into the `SupportedTypes` configuration.

Let me know if this is ok and I can help with a PR.

**Comment by fatkodima** [comment]
Solved this with `Layout/FirstArgumentIndentation` cop.

Now it looks
```ruby
foo = bar(
  1,
  2,
  3
)
```
which is also good for me.
